python/paddle/profiler/profiler.py

```diff
@@ -734,6 +749,7 @@ def summary(self,
                 sorted_by=SortedKeys.CPUTotal,
                 op_detail=True,
                 thread_sep=False,
+                views=None,
```

Comment:
In order to ensure compatibility, the new parameter are usually added at the end of API parameters  , but the `views` is not. do we have another considerations?